python/src/GaussianProcessFitterResult_doc.i.in

```diff
@@ -0,0 +1,194 @@
+metaModel : :class:`~openturns.Function`
+    The meta model: :math:`\tilde{\cM}: \Rset^{\inputDim} \rightarrow \Rset^{\outputDim}`, defined in :eq:metaModelGPF.
+residuals : :class:`~openturns.Point`
+    The residual errors.
+relativeErrors : :class:`~openturns.Point`
+    The relative errors.
+basis :  :class:`~openturns.Basis`
+    Functional basis of size :math:`b` : :math:`(\varphi^l: \Rset^{\inputDim} \rightarrow \Rset^{\outputDim})` for each :math:`l \in [1, b]`.
+    Its size should be equal to zero if the trend is not estimated.
+trendCoef : sequence of float
+    The trend coefficients vectors :math:`(\vect{\alpha}^1, \dots, \vect{\alpha}^{\outputDim})` stored as a Point.
+covarianceModel : :class:`~openturns.CovarianceModel`
+    Covariance function of the Gaussian process with its optimized parameters.
+optimalLogLikelihood : float
+    The maximum log-likelihood corresponding to the model.
+linAlgMethod : int
+    The used linear algebra method to fit the model.
+    0 for `LAPACK`, 1 for `HMAT`
+
+Notes
+-----
+The structure is usually created by the method *run()* of a :class:`~openturns.experimental.GaussianProcessFitter`, and obtained thanks to the *getResult()* method.
+
+The meta model :math:`\tilde{\cM}: \Rset^{\inputDim} \rightarrow \Rset^{\outputDim}` is defined by:
+
+.. math::
+    :label: metaModelGPF
+
+    \tilde{\cM}(\vect{x}) = \left(
+      \begin{array}{l}
+        \mu_1(\vect{x}) \\
+        \dots  \\
+        \mu_p(\vect{x}) 
+       \end{array}
+     \right)
+
+where :math:`\mu_l(\vect{x}) = \sum_{j=1}^{b} \alpha_j^l \varphi_j^l(\vect{x})` and :math:`\varphi_j^l: \Rset^{\inputDim} \rightarrow \Rset` are the trend functions 
+(the :math:`l-th` marginal of `\varphi(x)`).
+
+
+.. math::
+    :label: metaModelWithTGPF
+
+    \tilde{\cM}(\vect{x}) = \left(
+      \begin{array}{l}
+        \mu_1\circ T(\vect{x}) \\
+        \dots  \\
+        \mu_p\circ T(\vect{x}) 
+       \end{array}
+     \right)
+
+Examples
+--------
+Create the model :math:`\cM: \Rset \mapsto \Rset` and the samples:
+
+>>> import openturns as ot
+>>> from openturns.experimental import GaussianProcessFitter
+>>> f = ot.SymbolicFunction(['x'],  ['x * sin(x)'])
+>>> sampleX = [[1.0], [2.0], [3.0], [4.0], [5.0], [6.0]]
+>>> sampleY = f(sampleX)
+
+Create the algorithm:
+
+>>> basis = ot.Basis([ot.SymbolicFunction(['x'], ['x']), ot.SymbolicFunction(['x'], ['x^2'])])
+>>> covarianceModel = ot.GeneralizedExponential([2.0], 2.0)
+>>> algo = GaussianProcessFitter(sampleX, sampleY, covarianceModel, basis)
+>>> algo.run()
+
+Get the result:
+
+>>> result = algo.getResult()
+
+Get the meta model:
+
+>>> metaModel = result.getMetaModel()
+>>> graph = metaModel.draw(0.0, 7.0)
+>>> cloud = ot.Cloud(sampleX, sampleY)
+>>> cloud.setPointStyle('fcircle')
+>>> graph = ot.Graph()
+>>> graph.add(cloud)
+>>> graph.add(f.draw(0.0, 7.0))
+>>> graph.setColors(['black', 'blue', 'red'])
```

Comment:
I think that the default colors will manage this: there is no need to set the colors explicitly here.

Reply:
Ah it was implemented long time ago. I didn't check the behaviour without explicitely settings colors. Will do it